### PR TITLE
feat: add footer

### PR DESCRIPTION
**Pull request by DubichTonya**
1. Add footer.

### CODE REVIEW

File: src/app/core/components/footer/footer.component.html [hunk 1]
```diff
@@ -1 +1,20 @@
-<p>footer works!</p>
+<footer>
+  <ul class="page-footer__list">
+    <li *ngFor="let developer of teammates$ | async">
```
Comment (rss-maksim) [inline src/app/core/components/footer/footer.component.html]: Could please add ```trackBy```

File: src/app/core/components/footer/footer.component.html [hunk 2]
```diff
@@ -1 +1,20 @@
...
+      <a [href]="developer.github">
```
Comment (rss-maksim) [inline src/app/core/components/footer/footer.component.html:4]: ```target="_blank"```?

**Review by rss-maksim — APPROVED** [review]

**Review by IlyaKozak — APPROVED** [review]

File: src/app/core/components/footer/footer.component.html (hunk 1 above)
Comment (DubichTonya) [reply src/app/core/components/footer/footer.component.html]: Done

File: src/app/core/components/footer/footer.component.html (hunk 2 above)
Comment (DubichTonya) [reply src/app/core/components/footer/footer.component.html]: Done

**Pull request merged**